tests/integration_test.go

```diff
@@ -185,3 +201,20 @@ func TestDiffAndAnalysis(t *testing.T) {
 		})
 	}
 }
+
+func TestMain(m *testing.M) {
```

Comment:
I think this is the culprit

Reply:
good call, also turns out Docker actually wasn't available in our Travis runs. fixed